### PR TITLE
Add configurable cookie domain property

### LINKED ISSUE #19: Allow configurable cookie domain property
### Describe the feature

I have two subdomains:

- frontend.example.com (Nuxt)
- backend.example.com (Strapi)

and the problem is this - from frontend.example.com no cookies are passed to backend.example.com. I need to send sessionId to Strapi and save to database.

If you add the `domain: ".example.com"` property, then the cookies are sent as expected.

https://github.com/sidebase/nuxt-session/blob/5cbbad4b0a7b5c4d721f72b8507c007c10e67d5b/src/runtime/server/middleware/session/index.ts#L9-L18

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by blumgart**
Closes #19 .

### CODE REVIEW

File: src/module.ts [hunk 1]
```diff
@@ -44,6 +44,13 @@ declare interface SessionOptions {
     * @docs https://github.com/unjs/unstorage
     */
    storageOptions: CreateStorageOptions,
+   /**
+   * Set the cookie domain
```
Comment (BracketJohn) [inline src/module.ts:48]: Can you expand this a bit, eg, what does it mean if the domain for a cookie is set vs it being null?

Linking to a relevant part of MDN docs would also be useful here (:

Comment (blumgart) [reply src/module.ts]: Hi, @BracketJohn. Thanks for response. Added a fix, my mistake)

File: src/module.ts [hunk 2]
```diff
@@ -44,6 +44,14 @@ declare interface SessionOptions {
     * @docs https://github.com/unjs/unstorage
     */
    storageOptions: CreateStorageOptions,
+   /**
+   * Set the cookie domain for sharing on subdomains. If not include or value is null, the current domain will be set.
```
Comment (BracketJohn) [inline src/module.ts:48]: May I suggest:
```suggestion
   * Set the domain the session cookie will be receivable by. Setting `domain: null` results in setting the domain the cookie is initially set on. Specifying a domain will allow the domain and all its sub-domains.
```

File: src/module.ts (hunk 1 above)
Comment (BracketJohn) [reply src/module.ts]: No worries, thanks for iterating here with me (:

File: src/module.ts (hunk 2 above)
Comment (blumgart) [reply src/module.ts:48]: Sure 👍

**Review by BracketJohn — APPROVED** [review]

**Comment by BracketJohn** [comment]
Merged, thanks @blumgart for the issue + the awesome contribution ❤️ 

Here's the release: https://github.com/sidebase/nuxt-session/releases/tag/0.2.3

**Pull request merged**